plugins/modules/swim_workflow_manager.py

```diff
@@ -810,24 +810,30 @@ def get_device_id(self, params):
             no device or multiple devices match the criteria, it raises an exception.
         """
         device_id = None
-        response = self.dnac._exec(
-            family="devices",
-            function='get_device_list',
-            op_modifies=True,
-            params=params,
-        )
-        self.log("Received API response from 'get_device_list': {0}".format(str(response)), "DEBUG")
+        self.log(params)
```

Comment:
self.log("Fetching device ID with parameters: {0}".format(params), "INFO")

Reply:
updated the log